<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNewFileDlg</class>
 <widget class="QDialog" name="CNewFileDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新建文件</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_name">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>文件名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="styleSheet">
       <string notr="true">background:#606060;color:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_describe">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>描述</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_describe">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#303030;color:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>294</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>新建</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="styleSheet">
       <string notr="true">background:#303030;color:white</string>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="gridSize">
       <size>
        <width>60</width>
        <height>80</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <item>
       <property name="text">
        <string>Scene</string>
       </property>
       <property name="icon">
        <iconset resource="ui_res.qrc">
         <normaloff>:/picture/ui_picture/file/scene.png</normaloff>:/picture/ui_picture/file/scene.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Script</string>
       </property>
       <property name="icon">
        <iconset resource="ui_res.qrc">
         <normaloff>:/picture/ui_picture/file.png</normaloff>:/picture/ui_picture/file.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UI</string>
       </property>
       <property name="icon">
        <iconset resource="ui_res.qrc">
         <normaloff>:/picture/ui_picture/file/UI.png</normaloff>:/picture/ui_picture/file/UI.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Map</string>
       </property>
       <property name="icon">
        <iconset resource="ui_res.qrc">
         <normaloff>:/picture/ui_picture/objects/map.png</normaloff>:/picture/ui_picture/objects/map.png</iconset>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="ui_res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CNewFileDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
